### PR TITLE
f-string cleanup #6068

### PR DESCRIPTION
**Pull request by eve-johns**
resolves the f-string part of [#6068](https://github.com/dbt-labs/dbt-core/issues/6068)

### Description
Cleaned up multiline f string formatting issue.

### Checklist

- [x] I have read [the contributing guide](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md) and understand what's expected of me
- [x] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [x] I have run this code in development and it appears to resolve the stated issue
- [x] This PR includes tests, or tests are not required/relevant for this PR
- [x] I have [opened an issue to add/update docs](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose), or docs changes are not required/relevant for this PR
- [x] I have run `changie new` to [create a changelog entry](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md#adding-a-changelog-entry)

### CODE REVIEW

File: core/dbt/context/providers.py [hunk 1]
```diff
@@ -220,12 +220,12 @@ def _repack_args(self, name: str, package: Optional[str]) -> List[str]:
...
         if not isinstance(name, str):
             raise CompilationException(
-                f"The name argument to ref() must be a string, got " f"{type(name)}"
+                f"The name argument to ref() must be a string, got {type(name)}"
             )
 
         if package is not None and not isinstance(package, str):
             raise CompilationException(
-                f"The package argument to ref() must be a string or None, got " f"{type(package)}"
+                f"The package argument to ref() must be a string or None, got  {type(package)}"
```
Comment (emmyoop) [inline core/dbt/context/providers.py:228]: This has two spaces instead of one.

**Review by emmyoop — CHANGES_REQUESTED** [review]
One small space to remove and then it looks good once you add the changelog and run pre-commit!  Thanks for participating in the Hackathon at Coalesce!

File: core/dbt/context/providers.py (hunk 1 above)
Comment (eve-johns) [reply core/dbt/context/providers.py]: Thank you for pointing it out!

**Comment by emmyoop** [comment]
You'll want to run `pre-commit` ([docs](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md#pre-commit)) and also [create a Changelog entry](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md#adding-a-changelog-entry) (Under the Hood).

**Review by leahwicz — APPROVED** [review]

**Review by emmyoop — APPROVED** [review]

**Pull request merged**